misc/formatter/modules/formatter-core/src/test/java/org/ballerinalang/formatter/core/ParserTestFormatter.java

```diff
@@ -134,6 +134,8 @@ public void test(String fileName, String path) throws IOException {
                 "assignment_stmt_source_11.bal", "assignment_stmt_source_10.bal", "assignment_stmt_source_14.bal",
                 "match_stmt_source_09.bal", "match_stmt_source_13.bal", "match_stmt_source_04.bal",
                 "match_stmt_source_15.bal", "match_stmt_source_03.bal", "match_stmt_source_16.bal",
+                "match_stmt_source_06.bal", "match_stmt_source_07.bal", "match_stmt_source_11.bal",
```

Comment:
Any specific reason to disable these test cases? Since these we passing earlier, shall we have a look at it @IrushiL ?

Reply:
The above line was already there. I only replaced it from L156 to here. And only "match_stmt_source_19.bal" was added newly which is a new test case.

Reply:
"match_stmt_source_19.bal" also enabled with 1613179.

Reply:
+1